src/org/usfirst/frc2175/config/WiringConfig.java

```diff
@@ -25,6 +25,7 @@
     private Encoder leftDriveEncoder;
     private Encoder rightDriveEncoder;
     private Gyro gyro;
+    private DoubleSolenoid driveShifters;
```

Comment:
I can almost guarantee the shifters will be on a single solenoid. So that will be just the `Solenoid` class, not `DoubleSolenoid`.

Also, that means we only need one port. So you don't need to specify a and b.